strawberryfields/apps/qchem/vibronic.py

```diff
@@ -138,6 +140,55 @@ def energies(samples: list, w: np.ndarray, wp: np.ndarray) -> Union[list, float]
     return [np.dot(s[: len(s) // 2], wp) - np.dot(s[len(s) // 2 :], w) for s in samples]
 
 
+def VibronicTransition(modes: int):
```

Comment:
Will we also be changing the `evolution()` function to caps?

Reply:
Yes, there will be a PR for adding `TimeEvolution` function. We can discuss this further in the next PR.